src/test/clojure/clara/performance/test_compilation.clj

```diff
@@ -0,0 +1,154 @@
+(ns clara.performance.test-compilation
+  (:require [clojure.test :refer :all]
+            [clara.rules.compiler :as com]
+            [clara.rules :as r]
+            [clara.rules.durability.fressian :as dura])
+  (:import [java.io ByteArrayOutputStream ByteArrayInputStream]))
+
+(defn filter-fn
```

Comment:
I don't understand how this would work. `gensym` should return a new and unique symbols every time it is called. So this `filter-fn` will always be false.

If that is intentional, some sort of explanation doc string would be helpful.

Reply:
As these rules never fire, this is a placeholder.